browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -517,6 +517,7 @@ function _treelistboxControl(parentContainer, data, builder) {
 			tr.innerText = _('Headings and objects that you add to the document will appear here');
 		} else {
 			L.DomUtil.addClass(table, 'empty');
+			table.setAttribute('disabled', 'true');
```

Comment:
empty != disabled

this can confuse people later (developers)

eg. pivot table drag and drop: when one treeview is empty but should accept drop, it is empty but not disabled. while debugging someone will spend time to understand why it is disabled suddenly

could you instead apply proper styling to `.empty` treeview in CSS?